src/openforms/submissions/templates/submission_logs/rules.html

```diff
@@ -0,0 +1,18 @@
+{% if node.nodeType == 'operation' %}
+    <li class="rule-list"><b> {{node.operator}}: </b> </li>
+    <ul>
+{% else %}
+
+    <li class="rule-list">
+        <b>{{node.stepName}}: {{node.key}} &#40;{{node.label}}&#41;</b> <i>{{node.operator}}</i> <b> {{node.value}}</b> 
```

Comment:
Please put spaces in the curly braces

```suggestion
        <b>{{ node.stepName }}: {{ node.key }} &#40;{{ node.label }}&#41;</b> <i>{{ node.operator }}</i> <b> {{ node.value }}</b> 
```